src/acrcssc/azext_acrcssc/tests/latest/test_cssc_scenario.py

```diff
@@ -0,0 +1,39 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer, live_only
+
+
+class AcrcsscScenarioTest(ScenarioTest):
+    @live_only()
```

Comment:
May I ask why do you need to mark this test as ` @live_only()`?

Reply:
the end-2-end scenario needs to do authenticate and do a 'docker push' an artifact through a module (oras-py) to the registry being tested, test keeps failing when not in live mode. Could not get it to work without live_only(), but would appreciate if you have any recommendation or doc I can follow.
Added unit tests for all sections of the extension to properly test where I can mock the required calls (everything that is not extension code) if it makes any difference.

Reply:
OK, got it, thanks~